### PR TITLE
Keep nav element selected when in children pages

### PR DESCRIPTION
**Pull request by evmiguel**
This issue handles #107.

The expected behavior should be when a design pattern is clicked, the "Patterns" element in the nav should be active.

### CODE REVIEW

**Review by howard-e — APPROVED** [review]
LGTM!

An additional thought: Perhaps this is covered by another issue but seems like the same could be applied to the Practices & Index (Examples) pages. Although if this should be done, in the instance of Index, those subpaths can be reached from Patterns as well so not sure if this is worth the additional effort to determine the navigation path or keep it solely on Index.

**Comment by richnoah** [comment]
@howard-e you are correct.  @evmiguel this should also to /practices and when a visitor goes into a pattern, a pattern example or one of the practices.

**Comment by evmiguel** [comment]
> @howard-e you are correct. @evmiguel this should also to /practices and when a visitor goes into a pattern, a pattern example or one of the practices.

Got it. I'll add the other nav links! Thank you both.

File: scripts/pre-build/library/getTemplateBoilerplate.js [hunk 1]
```diff
@@ -57,6 +57,11 @@ const addBodyClass = ${JSON.stringify(addBodyClass)};
 const enableSidebar = ${JSON.stringify(enableSidebar)};
 if (addBodyClass) document.body.classList.add(addBodyClass);
 if (enableSidebar) document.body.classList.add('has-sidebar');
+
+// Make nav item "Patterns" active when on a patterns page
+if (window.location.pathname.includes('patterns')) {
+  document.querySelector('a[href*="patterns/"]').classList.add('active');
+}
```
Comment (alflennik) [inline scripts/pre-build/library/getTemplateBoilerplate.js:64]: Could we do something similar for the examples too? If we're in the example-index folder highlight the index item?

Comment (evmiguel) [reply scripts/pre-build/library/getTemplateBoilerplate.js:64]: Definitely. It makes sense to do include this, too.

**Review by alflennik — APPROVED** [review]
Great that you addressed the other page types too 👍

**Pull request merged**